pipelines/stripe_analytics/__init__.py

```diff
@@ -0,0 +1,90 @@
+""" This pipeline uses Stripe API and dlt to load data such as Customer, Subscription, Event and etc. to the database and to calculate the MRR and churn rate. """
+
+from typing import Any, Dict, Generator, Optional
+
+import dlt
+import stripe
+from dlt.common import pendulum
+from dlt.extract.source import DltResource
+from pendulum import DateTime
+
+from .helpers import pagination, transform_date
+from .metrics import calculate_mrr, churn_rate
+
+
+@dlt.source
+def incremental_stripe_source(
+    endpoints: list,
+    stripe_secret_key: str = dlt.secrets.value,
+    start_date: Optional[DateTime] = None,
+    end_date: Optional[DateTime] = None,
+) -> Generator[DltResource, Any, None]:
+    stripe.api_key = stripe_secret_key
```

Comment:
we need docstrings! your explanations from demo script should be also included here. and:
- what are valud endpoint names? point to the stripe docs? otherwise how people can guess what goes here?
- the start data is rather `initial_start_date` - it is only applied on first run. you must explain that
- what is the role of end date? you actually do not demonstrate usage 

no one is going to guess that you filter dates on created at. you may even put it in the name `initial_created_at`